huaweicloud/services/acceptance/fgs/data_source_huaweicloud_fgs_function_event_test.go

```diff
@@ -0,0 +1,90 @@
+package fgs
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func TestAccDataSourceFunctionEvent_basic(t *testing.T) {
+	var (
+		all = "data.huaweicloud_fgs_function_events.test"
+		dc  = acceptance.InitDataSourceCheck(all)
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acceptance.TestAccPreCheck(t)
+			acceptance.TestAccPreCheckFgsAgency(t)
```

Comment:
ditto

Reply:
seems this env is useless, because the function does not need the other services privileges.